tensorboard/webapp/metrics/store/metrics_selectors.ts

```diff
@@ -259,12 +259,24 @@ export const getCanCreateNewPins = createSelector(
 
 const selectSettings = createSelector(
   selectMetricsState,
-  (state): MetricsState['settings'] => state.settings
+  (state): MetricsSettings => {
+    return {
+      ...state.settings,
+      ...state.settingOverrides,
+    };
+  }
 );
 
 /**
  * Settings.
  */
+export const getMetricsSettingOverrides = createSelector(
+  selectMetricsState,
+  (state): MetricsState['settingOverrides'] => {
```

Comment:
I think `Partial<MetricsSettings>` as the return type would be more readable, to not require a jump to definition here.